components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -163,7 +165,7 @@ export default class PostMessageView extends React.PureComponent {
             siteURL: siteUrl,
             mentionKeys,
             atMentions: true,
-            channelNamesMap: getChannelsNameMapInCurrentTeam(store.getState()),
+            channelNamesMap: this.props.channelNamesMap,
```

Comment:
All of the rest of the props are being deconstructed up above so if you assign it there you use the object property value shorthand here.